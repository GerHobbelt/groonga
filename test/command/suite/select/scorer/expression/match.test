table_create Users TABLE_HASH_KEY ShortText
column_create Users tags COLUMN_VECTOR|WITH_WEIGHT ShortText

load --table Users
[
{"_key": "alice", "tags": {"wonder": 100}}
]

table_create Tags TABLE_HASH_KEY ShortText
column_create Tags users_tags COLUMN_INDEX|WITH_WEIGHT Users tags

select Users \
  --filter true \
  --scorer '_score = (tags @ "wonder")' \
  --output_columns _score
