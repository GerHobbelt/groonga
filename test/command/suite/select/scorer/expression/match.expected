table_create Users TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Users tags COLUMN_VECTOR|WITH_WEIGHT ShortText
[[0,0.0,0.0],true]
load --table Users
[
{"_key": "alice", "tags": {"wonder": 100}}
]
[[0,0.0,0.0],1]
table_create Tags TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Tags users_tags COLUMN_INDEX|WITH_WEIGHT Users tags
[[0,0.0,0.0],true]
select Users   --filter true   --scorer '_score = (tags @ "wonder")'   --output_columns _score
[[0,0.0,0.0],[[[1],[["_score","Int32"]],[1]]]]
